src/main/java/org/eclipse/golo/runtime/ClassReferenceSupport.java

```diff
@@ -55,7 +55,17 @@ public static CallSite bootstrap(MethodHandles.Lookup caller, String name, Metho
     try {
       return Class.forName(name, true, classLoader);
     } catch (ClassNotFoundException e) {
-      return null;
+      switch(name) {
```

Comment:
An other elegant approach is to use a `Map<String, Class>` and thus only `return PRIMITIVES.get(name)` instead of the `switch`, but I suspect it would be less efficient.